src/components/Collapse.js

```diff
@@ -21,19 +28,34 @@ const Collapse = ({ title, buttonStyle, buttonProps = {}, summaryStyle, detailsS
   }, [firstOpenRef, isOpened])
 
   return div(props, [
-    div({ style: { display: 'flex', alignItems: 'center', ...summaryStyle } }, [
+    div({
+      style: {
+        position: 'relative',
+        display: 'flex', alignItems: 'center',
+        ...summaryStyle
+      }
+    }, [
+      !titleFirst && angleIcon,
       h(Link, {
         'aria-expanded': isOpened,
         'aria-controls': isOpened ? id : undefined,
-        style: { display: 'flex', flex: 1, alignItems: 'center', marginBottom: '0.5rem', ...buttonStyle },
+        style: {
+          color: colors.dark(),
+          ...(noTitleWrap ? Style.noWrapEllipsis : {})
+        },
         onClick: () => setIsOpened(!isOpened),
-        ...buttonProps
+        hover,
+        tooltip,
+        tooltipDelay
       }, [
-        titleFirst && div({ style: { flexGrow: 1, ...(noTitleWrap ? Style.noWrapEllipsis : {}) } }, [title]),
-        angleIcon,
-        !titleFirst && title
+        div({
+          'aria-hidden': true,
+          style: { position: 'absolute', top: 0, left: 0, bottom: 0, right: 0, zIndex: 1 }
```

Comment:
`zIndex` usage almost always wants a comment. Why wasn't absolute positioning enough here?

Reply:
Added comments in https://github.com/DataBiosphere/terra-ui/pull/3240/commits/7ee084e2707f1222cb814c68200867588e327526. `zIndex: 1` lifts this element above sibling elements (such as `angleIcon`), so that clicking anywhere on the summary section will toggle the Collapse.